### PR TITLE
Set client engine to Android TV if SDK is used on AndroidTV

### PR DESCRIPTION
**Pull request by onufryk**

### CODE REVIEW

**Comment by mikeng13** [comment]
Please update the changelog also with the changes from the Java SDK and the AndroidTV engine. Also remember to Squash and Merge :)

**Comment by elliotykim** [comment]
@onufryk Yup, changelog and you can squash/merge.

This was already approved in phabricator, so no need for another approval, right? @mikeng13

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # Optimizely Android X SDK Changelog
+### 1.1.0
+February 17, 2017
+
+- Support Android TV SDK client engine
```
Comment (mikeng13) [inline CHANGELOG.md]: Also add that we updated to Java core 1.5 and maybe link to its changelogs.

Comment (onufryk) [inline CHANGELOG.md]: Thanks, great point

**Pull request merged**